Controllers/userController.js

```diff
@@ -149,13 +150,19 @@ const loginUser = async (req, res) => {
 //This logs the user out the app by removing the
 //Users token
 const logout = async (req, res) => {
+  const [id, username] = getToken.getToken(req);

```

Comment:
Once you fix the get token then you can use destructuring to get 
Whichever one you want/need